prosopopee/prosopopee.py

```diff
@@ -210,9 +210,7 @@ def ratio(self):
             + " -v error -select_streams v:0 -show_entries stream=width,height -of csv=p=0"
         )
         command_list = command.split()
-        # target is Type path.Path, encodes to bytes, decodes to str, which we can append to the
-        # list disgusting, I know. But it works
-        command_list.append(target.encode().decode())
+        command_list.append(str(target))
```

Comment:
I don't understand how this change is related to this PR, could you explain it '-'?

Reply:
Initially I've copy pasted the encode.decode to match the previous call but after exchanging on my issue str(path) will do the trick. So i've changed previous call to use the same.

Reply:
It'd probably make sense to have a different commit for this change alone since it's not related to fixing paths with spaces but rather a clean-up.

Reply:
ok done if you prefer like that.

Reply:
You need to remove this change from the original commit, not add a revert :) you probably need to force-push to your branch. Don't know how well it'll work with Github Pull Requests but we shall see :)